### PR TITLE
CompiledExceptionHandlerTest should not rely on profiling information.

### PR DESCRIPTION
**Pull request by rschatz**
This unit test sometimes randomly failed on travis, now it should be reliable.

### CODE REVIEW

File: graal/com.oracle.graal.compiler.common/src/com/oracle/graal/compiler/common/GraalOptions.java [hunk 1]
```diff
@@ -123,10 +123,10 @@
     public static final OptionValue<Boolean> DeoptALot = new OptionValue<>(false);
 
     @Option(help = "Stressed the code emitting explicit exception throwing code.", type = OptionType.Debug)
-    public static final StableOptionValue<Boolean> StressExplicitExceptionCode = new StableOptionValue<>(false);
+    public static final OptionValue<Boolean> StressExplicitExceptionCode = new OptionValue<>(false);
```
Comment (dougxc) [inline graal/com.oracle.graal.compiler.common/src/com/oracle/graal/compiler/common/GraalOptions.java]: The help message for this option and the one below should not be past tense. To be consistent with other options, it should be proscriptive: `help = "Stress the code ..."`

Comment (rschatz) [inline graal/com.oracle.graal.compiler.common/src/com/oracle/graal/compiler/common/GraalOptions.java]: Ok, I'll change that.

**Comment by tkrodriguez** [comment]
The fix looks fine but I'm not sure of the point of this test if we're simply bypassing the profiling information.  It's just testing the parser now instead of checking that a profiled exception throw properly trigger that logic in the parser.  Do we need a test for the latter?

**Comment by rschatz** [comment]
I interpreted these tests to check whether explicit exception edges work correctly. The assertion in `parseEager` makes sure that the parser did the correct thing, i.e. we're really testing the explicit exception case, but the main point of this test is that graal can sucessfully compile a graph that contains explicit exception edges.

**Comment by tkrodriguez** [comment]
I'm fine with that

**Comment by rschatz** [comment]
Fixed checkstyle error and squashed everything into a single commit.

**Pull request merged**